src/client.js

```diff
@@ -698,7 +698,7 @@ export class StreamChat {
 			},
 		);
 		this.connectionID = this.wsConnection.connectionID;
-		const cids = Object.keys(this.activeChannels || {});
+		const cids = Object.keys(this.activeChannels);
```

Comment:
what if `activeChannels` is null? It will throw exception in such case and client side code will stop working

Reply:
If it's missing, it will crash two lines below anyway (line 703).

According to the call path, it can't be null - always initialized.